language/documentation/book/translations/move-book-zh/Translators.md

```diff
@@ -1,30 +0,0 @@
-|    | 章节                       | 译者                                           | 校对                                                                                |
-|----|----------------------------|------------------------------------------------|-------------------------------------------------------------------------------------|
-| 0  | Intoduction                | Tom                                            | [@lshoo](https://github.com/lshoo)、[@Joe Chen](https://github.com/geometryolife)   |
```

Comment:
Is the deletion of this and other files intentional?

Reply:
Year, This is the translators info for the Chinese version.
I've rollup all the deleted files into one "Contribute".

Reply:
I don't think this file should be deleted. This is the list of contributors to move-book-zh from the original MoveDAO community. See “migrate Move Book of Chinese version from MoveDao” #380

Reply:
@geometryolife Actually, I've rollup the deleted files into one, the contribute.md,  with a more proper structure .

![image](https://user-images.githubusercontent.com/108982045/194859242-6ff2473d-641a-49cb-a069-65fe2d24385e.png)

Reply:
thanks❤️